src/filename_inspector.py

```diff
@@ -50,15 +70,12 @@ def find_non_matching_files(name_patterns, paths, excludes):
 
 def run():
     try:
-        name_patterns_raw = get_action_input(INPUT_NAME_PATTERNS)
-        name_patterns = name_patterns_raw.split(',') if name_patterns_raw else []
-        paths_raw = get_action_input(INPUT_PATHS)
-        paths = paths_raw.split(',')
-        excludes_raw = get_action_input(INPUT_EXCLUDES)
-        excludes = excludes_raw.split(',')
-        report_format = get_action_input(INPUT_REPORT_FORMAT).lower()
-        verbose_logging = get_action_input(INPUT_VERBOSE_LOGGING).lower() == 'true'
-        fail_on_violation = get_action_input(INPUT_FAIL_ON_VIOLATION).lower() == 'true'
+        name_patterns = get_action_list_input(INPUT_NAME_PATTERNS)
+        paths = get_action_list_input(INPUT_PATHS)
+        excludes = get_action_list_input(INPUT_EXCLUDES)
+        report_format = get_action_input(INPUT_REPORT_FORMAT)
+        verbose_logging = get_action_input(INPUT_VERBOSE_LOGGING) == 'true'
+        fail_on_violation = get_action_input(INPUT_FAIL_ON_VIOLATION) == 'true'
```

Comment:
Woudn't this fail on `True`?
What about either `.lower()` before comparison or converting to `bool`?

Reply:
I simplified that logic in commit 5c75aaf41f0445cb5ae00111fe449e871eb1a574.

Reply:
`get_action_input` has indeed changed, but this will still return set false if env var
`INPUT_FAIL_ON_VIOLATION` or `INPUT_VERBOSE_LOGGING` were set to e.g. `True` instead of `true`.

Maybe it is ok like that, just checking, if you don't want to react to different casings of true as well (that is my original point). If not, fine by me.

Reply:
No, it is not ok.
The action logic expects lowercase values. Thanks for the catch.

I added a default lower method call.

Reply:
Cool, this is what I had in mind. 👍